### ISSUE TITLE
[Bug]: microsoft-edge-dev cookie file path should be dev instead of Dev?

### ISSUE THREAD

**Issue by wangwb98** [thewh1teagle/rookie#19]
### What happened?

Hello,
I just found rookiepy says cannot find my microsoft edge browser's cookie. Then I digged a little bit and found it's searching for cookie file in ~/.config/microsoft-edge-Dev, but the edge browser (installed by Archlinux AUR package microsoft-edge-dev-bin) is using lower case dev, i.e ~/.config/microsoft-edge-dev.

After I created a synmbolic link microsoft-edge-Dev to microsoft-edge-dev, it's working greatly now.

Not sure if it's a bug worth fixing, just list it here in case other people met same issue.
Thanks for this nice package BTW.

### Steps to reproduce

Archlinux, install AUR package  microsoft-edge-dev-bin.
Run msedge, and double check ~/.config/microsoft-edge-dev is created.
use rookiepy to extract edge cookie, it reports error:
~~~
thread '<unnamed>' panicked at bindings/python/src/lib.rs:67:41:                                                                                                                              called `Result::unwrap()` on an `Err` value: can't find any cookies file                                                                                                                      note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Traceback (most recent call last):
  File "<string>", line 1, in <module>                                                         
pyo3_runtime.PanicException: called `Result::unwrap()` on an `Err` value: can't find any cookies file
~~~

### What browsers are you seeing the problem on?

Edge

### Relevant log output

```shell
thread '<unnamed>' panicked at bindings/python/src/lib.rs:67:41:                                                                                                                              called `Result::unwrap()` on an `Err` value: can't find any cookies file                                                                                                                      note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Traceback (most recent call last):
  File "<string>", line 1, in <module>                                                         
pyo3_runtime.PanicException: called `Result::unwrap()` on an `Err` value: can't find any cookies file
```

**Comment by thewh1teagle** [comment]
Thanks for report! Looks like a typo in config.rs file

**Comment by thewh1teagle** [comment]
Fixed in 48172529e0799114294fb35538f3b503a61470cd
